apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -129,7 +129,7 @@ export class AppRoomsConverter {
 
 		const departmentId = await this.__getDepartment(room);
 
-		const servedBy = await this.__getUserIdAndUsername(room.servedBy);
+		const servedBy = await this.__getUserIdAndUsername(room?.servedBy?.id);
```

Comment:
`room` doesn't need to be checked.

```suggestion
		const servedBy = await this.__getUserIdAndUsername(room.servedBy?.id);
```

I wonder if we should keep it as `room.servedBy` and check if not `null` within `__getUserIdAndUsername`

Reply:
I wanted to keep the changes minimal, and since func was expecting a uid, i passed a uid.

But guess any way it's good 👀 (function before this changes it's already at 33 CC iirc)